demo/server.js

```diff
@@ -151,3 +152,28 @@ initSqlJs().then(SQL => {
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
 });
+
+ // Add a new accuracy record for a model
+  function addAccuracyRecord(accuracy)
+  {
+   db.run('INSERT INTO accuracy VALUES (?, ?, ?, ?);', [
+     accuracy.transactionHash,
+     accuracy.blockNumber,
+     accuracy.modelId,
+     accuracy.accuracy,
+   ]);
+   fs.writeFile(dbPath, Buffer.from(db.export()), () => { });
+  }
+
+ // Get the accuracy history for a model
+ function getAccuracyHistory(modelId)
+ {
+  const getAccuracyStmt = db.prepare('SELECT * FROM accuracy WHERE model_id == $modelId ORDER BY timestamp;');
+  const result = getAccuracyStmt.getAsObject({ $modelId: modelId });
```

Comment:
![image](https://user-images.githubusercontent.com/56452820/117570875-d4439700-b0e9-11eb-800e-f13a42b9a380.png)
client side results after running

Reply:
![image](https://user-images.githubusercontent.com/56452820/117571044-7fece700-b0ea-11eb-97e4-4f00c26bc718.png)
heath api also running !

Reply:
![image](https://user-images.githubusercontent.com/56452820/117571145-cb06fa00-b0ea-11eb-9b4e-90c0b12f59d9.png)
all models i found ! with get all

Reply:
So the existing methods work, that's not surprising since they haven't changed. What about your new method, it would be good to see that you can actual get the history of accuracies from your new table because the documentation for `getAsObject` makes it seem like the current implementation will not work.

Reply:
> So the existing methods work, that's not surprising since they haven't changed. What about your new method, it would be good to see that you can actual get the history of accuracies from your new table because the documentation for `getAsObject` makes it seem like the current implementation will not work.

I was going to talk about it, I created a function to add data in the accuracy table but had not called it anywhere so no new data is being added in the accuracy table. We are having 3 models. where should I call this function (addAccuracyHistory) so that its data gets added to the accuracy table

Reply:
You can start by making an endpoint, `app.post('/api/accuracy', jsonParser, (req, res) => { ...` and calling you new method there.
Also make this following to retrieve the accuracy history:
```JavaScript
app.get('/api/accuracy', (req, res) => {
    const { modelId, address } = req.query;
...
```

As for where to use those, we can do that the client/ folder but we'll do that in another pull request.

Reply:
![image](https://user-images.githubusercontent.com/56452820/117583402-91eb7b80-b124-11eb-93a1-2a33f4ed7e2f.png)
This is the get API  i created ,
if db is already created then i have to run the  add table accuracy command in if to add new table , because it will load the previous db which don't have accuracy table. I have added command in else block to create new db

Reply:
![image](https://user-images.githubusercontent.com/56452820/117584064-433fe080-b128-11eb-882e-b5382b0f5dfc.png)
`getAsObject` is giving only one object of the table , I had made the result on console

Reply:
Right, so the `getAccuracyHistory` method is not very useful as it is because its name is misleading and it only gets one row.

Reply:
Yeah, we can do all this in one get Api